scss/_patterns_code-numbered.scss

```diff
@@ -11,6 +11,7 @@ $sidebar-width: 4.5rem !default;
     display: inline-block;
     padding: $spv-inner--small $sph-inner 0 ($sidebar-width + $sph-inner);
     position: relative;
+    white-space: pre-wrap;
```

Comment:
Do we want to wrap by default?

Even if we do, this PR doesn't seem to be about updating old components, so it shouldn't be part of it.

Reply:
I think we should, for reasons mentioned [above](https://github.com/canonical-web-and-design/vanilla-framework/pull/3474#issuecomment-756071065), but you're right, I'll move it out of this PR.

Reply:
Ok - in this case the standard (not numbered) code block should do this as well by default (if it doesn't).

Also, my small concern is that we have a 'feature' (wrapping) as a default, so now, we will need to have a 'negative' modifier (`is-not-wrapping` ?) to disable it. This can of course be worked around with some good naming.